### PR TITLE
Lognormal fix

### PR DESCRIPTION
**Pull request by petrelharp**
Here's the fix I proposed in #1308.

### CODE REVIEW

File: stdpopsim/catalog/DroMel/dfes.py [hunk 1]
```diff
@@ -89,15 +89,18 @@ def _RagsdaleDFE():
...
     scale_negative = 5.42 / 2 / Ne
     shape_negative = 3.36
     h = 0.5
 
     synonymous = stdpopsim.MutationType()
     negative = stdpopsim.MutationType(
         dominance_coeff=h,
-        distribution_type="l",  # log-normal distribution
-        distribution_args=[-scale_negative, shape_negative],
+        distribution_type="ln",  # log-normal distribution
```
Comment (andrewkern) [inline stdpopsim/catalog/DroMel/dfes.py:102]:
```suggestion
        distribution_type="ln",  # negative log-normal distribution
```

Comment (andrewkern) [reply stdpopsim/catalog/DroMel/dfes.py:102]: or maybe "log-normal negative..."

File: stdpopsim/dfe.py [hunk 2]
```diff
@@ -23,7 +23,12 @@ class MutationType(object):
     - ``g``: gamma, two parameters (mean, shape)
     - ``n``: normal, two parameters (mean, SD)
     - ``w``: Weibull, two parameters (scale, shape)
-    - ``l``: logNormal, two parameters (mean and sd on log scale; see rlnorm)
+    - ``lp``: positive logNormal, two parameters (mean and sd on log scale; see rlnorm)
+    - ``ln``: negative logNormal, two parameters (mean and sd on log scale; see rlnorm)
+
+    Type "lp" is always positive, and type "ln" is the same as "lp", multiplied by -1.
```
Comment (andrewkern) [inline stdpopsim/dfe.py:29]:
```suggestion
    Type "lp" is always positive, and type "ln" is always negative. Both are the same log-normal, just reflected about zero
```

Comment (andrewkern) [reply stdpopsim/dfe.py:29]: trying to unpack this a bit

**Review by andrewkern — CHANGES_REQUESTED** [review]
is this still missing the `lp` distribution definition? I don't seem to see it here?

**Comment by petrelharp** [comment]
> is this still missing the `lp` distribution definition? I don't seem to see it here?

Not sure what you mean - it's implemented [here](https://github.com/popsim-consortium/stdpopsim/pull/1309/files#diff-cb24342fdfba499cf83efa2df3a903540da15727db67aafe2d87bf28af607bbdR125)?

File: stdpopsim/dfe.py (hunk 2 above)
Comment (petrelharp) [reply stdpopsim/dfe.py]: I've changed it; see what you think?

**Comment by petrelharp** [comment]
Heh, found a bug that I wasn't actually doing `lp`; now testing that `lp` is >0 and `ln` is <0.

File: stdpopsim/dfe.py (hunk 2 above)
Comment (andrewkern) [reply stdpopsim/dfe.py]: nice. much better.

**Review by andrewkern — APPROVED** [review]
LGTM @petrelharp

**Comment by andrewkern** [comment]
huh what's this failed test?

**Comment by petrelharp** [comment]
> huh what's this failed test? thinking it's just stochastic from that one slimulation

Hmph, yes, probably but I want to fix that test - that's why I found the last bug. (and! it shouldn't be stochastic!)

**Comment by petrelharp** [comment]
Ah, I hadn't set a seed on that one.

**Comment by petrelharp** [comment]
Ok, this should be ready to merge.

**Pull request merged**